### DISCUSSION TITLE
导演可以整个服务的部署文档么

### DISCUSSION THREAD

**Discussion by janily** [baizhiheizi/prsdigg#1114] [Q&A] [answered]
导演可以整个服务的部署文档么，想整下本地跑下，基于这个来耍耍其它的东东

**Comment by an-lee** [comment]
好，我抽时间整理下。

**Reply by janily** [reply]
导演，整整文档呀 😁

**Reply by an-lee** [reply]
好，这两天就整。

就是从零开始在本地把程序跑起来是吧。

**Reply by janily** [reply]
嗯嗯

**Comment by an-lee** [comment, marked as answer]
写了一个大致的步骤： https://github.com/baizhiheizi/quill/blob/main/CONTRIBUTING.md

可以先试试看，如果卡住在哪一步了，我再补充
